backend/apps/matching/urls.py

```diff
@@ -44,8 +44,23 @@
     path("map", views.map_view, name="map"),
     path("<p:p_type>/participant_JSON", views.map_JSON, name="participant_JSON"),
     ####################################
-    #  List view
+    #  List view / filtering
     ####################################
+    path(
+        "api/<p:p_type>/info/list",
```

Comment:
An endpoint where you can get all of our data sounds like a bad idea

Reply:
Same as we had before. We need to be able to retrieve the results.
As Freya sad above this is a workable mockup and since we are still working on a release ready product there is no need that every PR needs to be feature complete. The idea was that it should be easier to review this way. If we include all follow-up changes in this PR it will be enormous, which was what you criticized with my big PRs.

Next step will also be to add an obligatory location and distance filter to this api call. But we can never prevent scraping of all publicly accessible data. It might be a good idea to implement a rate limiting of some sort to make this harder. Or an enforced delay between queries, this would be a new issue

Reply:
When is 

> before

for you?

> In m4h an attacker had to hop through Germany by repeatedly changing their location by 50km (?) to copy all of our data, now it's just an API call 😬

Reply:
This point is really not about having a release ready product but about not introducing an even bigger vulnerability

Reply:
Discussed in #177